components/content-section/server.css

```
@@ -66,13 +66,25 @@
     }
   }
 
+  pre {
+    padding: 1rem;
```

Nit: We seem to be using `em` below (in the `code {}` block):

```suggestion
    padding: 1em;
```